share/completions/brew.fish

```diff
@@ -1,15 +1,56 @@
+function __fish_brew_get_cmd
+  for c in (commandline -opc)
+    if [ (echo $c | string sub -l 1) != "-" ]
+      echo $c
```

Comment:
You'll probably want to break here to get just the first non-option.

Either that or I'm misunderstanding this and you can replace the entire loop with `string match -v -- '-*' (commandline -opc)`.

Note that `commandline -opc` will also contain `brew`.